service/resource/certificate/desired_test.go

```diff
@@ -1,9 +1,339 @@
 package certificate
 
 import (
+	"context"
+	"reflect"
 	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/spf13/afero"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/pkg/api/v1"
+
+	"github.com/giantswarm/micrologger/microloggertest"
```

Comment:
The separate block here is not necessary. We are not doing this in other projects.